### DISCUSSION TITLE
NgAlert doesn't send screenshots in alerts due to timeout

### DISCUSSION THREAD

**Discussion by apollo13** [grafana/grafana#53755] [Alerting]
**What happened**:

Our alerts are getting sent without screenshots because rendering takes to long.

**What you expected to happen**:

A way to configure the rendering timeout. Currently it seems to be hardcoded to 10 seconds:
https://github.com/grafana/grafana/blob/2b5d12310a4eb0f0230f64ed98b1faa138d99b7e/pkg/services/ngalert/image/service.go#L21

We are working on getting the results faster, but I wonder if it were possible to make that configurable.

**How to reproduce it (as minimally and precisely as possible)**:

Create an alert that links a dashboard & panel so the alert message includes an image. The panel in question needs to have a long loading time.

**Anything else we need to know?**:

**Environment**:
- Grafana version: 9.2.0 (latest docker image from a day ago)
- Data source type & version: MySQL
- OS Grafana is installed on: Centos7 with docker
- User OS & Browser: Fedora 36 with Firefox 
- Grafana plugins:
- Others:

**Comment by mellieA** [comment]
There is a discussion in https://github.com/grafana/grafana/issues/58687 if you would like to contribute

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
